style.css

```diff
@@ -446,3 +538,320 @@ textarea:focus {
   outline: none;
 }
 
+@media only screen and (min-width: 768px) {
+  /* for desktop */
+  body {
+    max-width: 1440px;
+  }
+
```

Comment:
- [ ] Great job implementing the desktop version of the portfolio, however, I would recommend to not making the width a fixed value, as one of the goals of the project is to learn responsive design, currently, as we have a fixed max-width the page is not responsive for large screens please see :point_down: 
![image](https://user-images.githubusercontent.com/97521557/230733852-b24d14ce-b55e-4838-8279-3ea2d3360188.png)

you can either loose this width property or use percentage like(100%) so that it will be responsive :+1: